### DISCUSSION TITLE
Is it possible to override the best guess when automating?

### DISCUSSION THREAD

**Discussion by EmptyJay** [jkwill87/mnamer#121] [Questions] [answered]
I discovered that my episodes of "9-1-1: Lone Star" were renamed to "Lone Star Law" by mistake. Is it possible to specify that a certain filename corresponds with a specific series? I'd rather not make special cases in my automation. I have a recollection of being able to do something like this in the tvname config file; match a pattern and specify the behavior.

Thanks!

**Comment by jkwill87** [comment, marked as answer]
The problem here is that guessit, which mnamer uses to parse metadata from file names, seems to be under the impression that `911` is part of the episode information and strips it out from the series name:
```
$ python -m guessit .\demo\911.lone.star.s01e01.mp4
For: .\demo\911.lone.star.s01e01.mp4
GuessIt found: {
    "absolute_episode": 911,
    "title": "lone star",
    "season": 1,
    "episode": 1,
    "container": "mp4",
    "mimetype": "video/mp4",
    "type": "episode"
}
```

Its kind of hacky but we can add it back using the [replace_before](https://github.com/jkwill87/mnamer/wiki/Settings#config-only-settings) config setting which replaces metadata before it gets passed to TvMaze:

``` json
    // ...
    "replace_before": {
        "Lone Star": "911 Lone Star"
    },
    // ...
```

![image](https://user-images.githubusercontent.com/4343678/106228099-aa103e00-61b8-11eb-9355-6eb6b2c3fd58.png)

**Reply by EmptyJay** [reply]
Thank you. That's exactly what I was looking for. I browsed the documentation to see if anything was mentioned, but I didn't dig into the depths of the config file.
